Source/JavaScriptCore/bytecompiler/NodesCodegen.cpp

```diff
@@ -649,8 +654,9 @@ RegisterID* PropertyListNode::emitBytecode(BytecodeGenerator& generator, Registe
             continue;
         }
 
-        if (p->isStaticClassField()) {
+        if (p->isStaticClassElement()) {
             ASSERT(staticFieldLocations);
+            dataLogIf(Options::useDebugLog(), "PropertyListNode::emitBytecode isStaticClassElement 1\n");
             staticFieldLocations->append(p->position());
             continue;
         }
```

Comment:
Here we need to add the positions of static block to `staticFieldLocations` for initializer evaluation.